### DISCUSSION TITLE
Help! - M1 MacBook Air - BigSur - "Could not install python3 as default"

### DISCUSSION THREAD

**Discussion by mplogace** [Homebrew/discussions#1378] [Getting started] [answered]
Dear Community,

I have tried to install python 3.9.2 to my M1 Mac, and I could not find it when I type "Python" in the terminal, which is showing it's still in python 2.7 (default version comes with Mac for compatibility)
I looked into youtube videos, (homebrew, code etc.,) but I'm facing an issue with that.

I'm a beginner, can anyone help me on this matter

Thanks in advance!!!

**Comment by carlocab** [comment, marked as answer]
You need to call Homebrew Python3 as `python3`.

If you'd like to call it as `python`, see the message shown after you did `brew install python@3.9`. (You can see it again by doing `brew info python@3.9`.)

**Reply by mplogace** [reply]
Could you please help me with installing Python 3? 
when I enter "python"
```
~ % python

WARNING: Python 2.7 is not recommended. 
This version is included in macOS for compatibility with legacy software. 
Future versions of macOS will not include Python 2.7. 
Instead, it is recommended that you transition to using 'python3' from within Terminal.
```

when I enter "python3"

```
~ % python3
Python 3.9.2 (v3.9.2:1a79785e3e, Feb 19 2021, 09:06:10) 
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```
then I installed:

`brew install python3` showed me:

```
==> python@3.9
Python has been installed as
  /opt/homebrew/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /opt/homebrew/opt/python@3.9/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /opt/homebrew/lib/python3.9/site-packages

tkinter is no longer included with this formula, but it is available separately:
  brew install python-tk@3.9

See: https://docs.brew.sh/Homebrew-and-Python
```

as you see, "I have no idea about /opt/ directory, it's supposed to be /usr/

Please do help me!

**Reply by carlocab** [reply]
`/opt/homebrew` is the default Homebrew prefix on Apple M1, so you definitely want your install to be in `/opt/homebrew`.

You have two alternatives here:

1. Always refer to Python3 as `python3` rather than `python`. This is the recommended way to use Homebrew Python3.
2. Add `/opt/homebrew/opt/python@3.9/libexec/bin` to your `PATH`. This will allow you to call Homebrew Python3 as `python`. (If you're not sure how to do this, you can Google "unix PATH" or "macos PATH". There should be plenty of material online that explains what `PATH` is and how to use it.)

**Reply by mplogace** [reply]
Thank you so much! you saved my life!

```
~ % python
Python 3.9.4 (default, Apr  4 2021, 17:42:23) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```
